apps/trackers/common.py

```diff
@@ -116,22 +116,37 @@ def _summary_prefixes(self) -> str:
         if self.tracker.is_embargoed:
             prefixes.append("EMBARGOED")
 
+        # no incident priority has been given
+        # thus let us assume there is none
+
         if any(
             flaw
             for flaw in self.flaws
             if flaw.major_incident_state == Flaw.FlawMajorIncident.APPROVED
         ):
             prefixes.append("[Major Incident]")
 
-        # we can theoretically have normal and CISA one at once
-        # and then we prioritize the normal one above CISA
         elif any(
             flaw
             for flaw in self.flaws
             if flaw.major_incident_state == Flaw.FlawMajorIncident.CISA_APPROVED
         ):
             prefixes.append("[CISA Major Incident]")
 
+        elif any(
+            flaw
+            for flaw in self.flaws
+            if flaw.major_incident_state == Flaw.FlawMajorIncident.MINOR
+        ):
+            prefixes.append("[Minor Incident]")
+
+        elif any(
+            flaw
+            for flaw in self.flaws
+            if flaw.major_incident_state == Flaw.FlawMajorIncident.ZERO_DAY
+        ):
+            prefixes.append("[0-day]")
+
```

Comment:
Could this be rewritten to reduce the O(n) complexity? if the flaw is `ZERO_DAY` we have to loop the entire flaw list at least 3 times.

This is my suggestion (could be completely wrong 😆 ): 
```python
prefix_map = {
    Flaw.FlawMajorIncident.APPROVED: "[Major Incident]",
    Flaw.FlawMajorIncident.CISA_APPROVED: "[CISA Major Incident]",
    Flaw.FlawMajorIncident.MINOR: "[Minor Incident]",
    Flaw.FlawMajorIncident.ZERO_DAY: "[0-day]"
}

for flaw in sorted(self.flaws, key=lambda flaw: flaw.major_incident_state):
    if flaw.major_incident_state in prefix_map:
        prefixes.append(prefix_map[flaw.major_incident_state])
        break
```

Reply:
Oh sure! This way is really silly :smile: silly copy-pasting

Reply:
And good suggestion! I will use it. Thank you!